Core/Core/Grades/ViewModel/GradeListViewModel.swift

```diff
@@ -82,20 +56,7 @@ public final class GradeListViewModel: ObservableObject {
     }
 
     // MARK: - Private properties
-    private var lastKnownDataState: GradeListData? {
-        didSet {
-            if !isInitialGradingPeriodSet {
-                isInitialGradingPeriodSet = true
-                gradeHeaderIsVisible = false
-                state = .initialLoading
-                let id = getSelectedGradingPeriodId(
-                    currentGradingPeriodID: lastKnownDataState?.currentGradingPeriodID,
-                    gradingPeriods: lastKnownDataState?.gradingPeriods ?? []
-                )
-                selectedGradingPeriod.accept(id)
-            }
-        }
-    }
+    private var lastKnownDataState: GradeListData?
```

Comment:
I deleted this code and called it after the grading period finished to avoid showing the grade title, but the loader still appears.